Upgrading mysql extension in insider gallery

nitishsharma25 opened the pull request:
This PR fixes #
Upgrade the MySQL Extension in insider gallery.

VSIX file: https://github.com/microsoft/azuredatastudio-mysql/releases/download/v0.2.5/azuredatastudio-mysql-0.2.5.vsix

alanrenmsft commented:
@nitishsharma25 , FYI, this is not insider gallery, you are updating the stable gallery.

nitishsharma25 commented:
> @nitishsharma25 , FYI, this is not insider gallery, you are updating the stable gallery.

@alanrenmsft, removed the change from stable and added in insider gallery.
